.github/workflows/c-cpp.yml

```diff
@@ -13,11 +13,12 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: update
-      run: sudo apt update
-    - name: install packages
-      run: sudo apt-get install --no-install-recommends -y debhelper-compat devscripts libapt-pkg-dev gettext libgtk-3-dev libvte-2.91-dev intltool xmlto sharutils libept-dev
-      # libsm-dev lsb-release (already installed on image)
+    - name: update and install packages
+      run: |
+        sudo apt update
+        sudo apt build-dep -y ./
+        sudo apt install devscripts -y
```

Comment:
I assume this is needed for having the `debuild` command? If so, maybe you could add a small comment?

Reply:
done :)